vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2163,6 +2169,16 @@ private void launchInvitePeople() {
         }
     }
 
+    /**
+     * Launch audio recorder intent
+     */
+    @SuppressLint("NewApi")
```

Comment:
Why did you add this "SuppressLint" ?

Reply:
Sorry it happened during copy-paste. Will remove